EIPS/eip-7742.md

```diff
@@ -33,67 +33,81 @@ This EIP also changes how the EL sources the current blob target value for two r
+
+#### Updated Fee calculations
+
+With the changing target, we also need to scale the `BLOB_BASE_FEE_UPDATE_FRACTION` from [EIP-4844](./eip-4844.md) accordingly to bound the price jumps by +-12.5%. But this introduces some irregularity on the fork block because excess blob gas is effectively scaled down by this changed factor in the fee calculations.
+
+However we consider the target change events to be few and far between and hence not a cause of concern. The likely effect of this factor update will be to bring down the base fee on the fork block and hence would not negatively impact the transactions in txpool.
 
 ## Specification
 
-### Block structure and validity
+### Block processing
 
 Upon activation of this EIP, execution clients **MUST** extend the header schema with an
-additional 64-bit field: the `target_blobs_per_block`. This value is set to the current target blob count. The Engine API
-is modified along with this EIP to provide the `target_blobs_per_block` with each payload and implementations can use this
-value to correctly set the block header field.
+additional 64-bit field: the `target_blobs_per_block`. This values is set to the one provided by the CL during block production.
 
-Validity of the `target_blobs_per_block` is guaranteed from the consensus layer, much like how withdrawals are handled.
+Any reference to `TARGET_BLOB_GAS_PER_BLOCK` from [EIP-4844](./eip-4844.md) (for e.g. for `excess_blob_gas` calculations) can be derived by taking the `target_blobs_per_block` from the EL block header and multiplying by `GAS_PER_BLOB`.
 
-When verifying a block, execution clients **MUST** ensure the target blob count in the block header matches the one
-provided by the consensus client.
+Validity of these values is guaranteed from the consensus layer, much like how withdrawals are handled. Hence when verifying a block, execution clients **MUST** ensure the target blobs per block in the block header matches the one provided by the consensus client.
 
-For a genesis block with no existing parent, the value should be set according to the agreed specification for the
-target blob count given by that genesis block's protocol rule set.
+Furthermore `get_base_fee_per_blob_gas` as specified by [EIP-4844](./eip-4844.md) is as modified below:
 
-### Block processing
+| Constant | Value |
+| - | - |
+| `BLOB_BASE_FEE_UPDATE_FRACTION_PER_TARGET_BLOB` | `1112825` |
+
+
+```python
+def calc_excess_blob_gas(parent: Header) -> int:
+  parent_target_blob_gas = parent.target_blobs_per_block * GAS_PER_BLOB
+  if parent.excess_blob_gas + parent.blob_gas_used < parent_target_blob_gas:
```

Comment:
These 4 lines can be replaced with

`return max(parent.excess_blob_gas + parent.blob_gas_used  - parent_target_blob_gas, 0)`

It also more intuitively captures the excess as a non-zero accumulation of `error=parent.blob_gas_used  - parent_target_blob_gas`

Reply:
i guess we can leave it as it is inline with 4844 style